packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -13,6 +13,7 @@ interface StyledCheckboxProps extends ColorProps {
   onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
   label?: string;
   feedbackColor?: ThemeFeedbackColor;
+  colorScheme: 'light' | 'dark';
```

Comment:
Die hier bitte in einen eigenen Type auslagern und Prop optional machen, inkl. default Prop

Reply:
Ausserdem bitte "variant" als Konvention nutzen statt colorScheme.